Neos.Neos/composer.json

```diff
@@ -13,6 +13,8 @@
         "neos/twitter-bootstrap": "*",
         "neos/content-repository": "*",
         "neos/fusion": "*",
+        "neos/fusion-afx": "*",
```

Comment:
Didn't we say that this can be removed when we merge it for version 5.1

@mficzel

Reply:
@markusguenther no this can be removed from composer json of the neos base distribution